lxd/device/device_utils_generic.go

```diff
@@ -12,7 +12,7 @@ func deviceJoinPath(parts ...string) string {
 	return strings.Join(parts, ".")
 }
 
-// validatePCIDevice returns whether a configured PCI device exists. It also returns true, if no device
+// validatePCIDevice returns whether a configured PCI device exists. It also returns nil, if no device
```

Comment:
"if no device" would be better as "if empty address supplied"

Reply:
True, I have rebased the commit with an updated description.